config/initializers/cors.rb

```diff
@@ -0,0 +1,19 @@
+# Be sure to restart your server when you modify this file.
+
+# Enable CORS for API v0 (logging is only activated when debug is enabled)
+debug_cors = ENV["DEBUG_CORS"].present? ? true : false
```

Comment:
Did Rails add it like this? `present?` already returns a boolean, so no need for the ternary.

Reply:
Nope, I just copy pasted from our code from 

https://github.com/thepracticaldev/dev.to/blob/5e3f645d299f4e583b11c9322cebc14b414404c3/config/application.rb#L55

:D 

I'll fix it eventually!